### PR TITLE
[KOGITO-835] - Create app/script to validate image.yaml file descript…

### PR DESCRIPTION
**Pull request by spolti**
…ors as part of CI jobs

Signed-off-by: spolti <fspolti@redhat.com>

### CODE REVIEW

**Comment by spolti** [comment]
Once we have this PR approved and merged, I'll generated a native binary and release it so we can use it on CI.

File: cekit-helpers/cekit-image-validator/src/main/java/org/kie/cekit/image/validator/commands/MultipleFilesValidator.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+            for (Path file : files) {
+                int returnCode = sfm.validate(file);
+                if (returnCode != 0) {
+                    return returnCode;
+                }
+            }
+
+        } catch (IOException e) {
+            e.printStackTrace();
```
Comment (sutaakar) [inline cekit-helpers/cekit-image-validator/src/main/java/org/kie/cekit/image/validator/commands/MultipleFilesValidator.java:47]:
```suggestion
            log.severe(e.getMessage());
```

File: cekit-helpers/cekit-image-descriptors/README.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# CeKit Image Descriptors
+
+This project only provides all the needed POJOs to parase CeKit yaml files.
```
Comment (radtriste) [inline cekit-helpers/cekit-image-descriptors/README.md:3]: parase ?

Comment (spolti) [reply cekit-helpers/cekit-image-descriptors/README.md:3]: parse**, lol

**Pull request merged**